test/BenchmarkTestCases.benchmark.mjs

```diff
@@ -577,34 +653,15 @@ async function registerSuite(bench, test, baselines) {
 						bench.add(
 							benchName,
 							async () => {
-								await new Promise((resolve, reject) => {
+								if (GENERATE_PROFILE) {
+									await withProfiling(benchName, () =>
+										runWebpack(webpack, config)
+									);
```

Comment:
Can we do it for watching too? Will be great to see some our benches are unstable for watch

Reply:
I can add it for watch initialization, but because watch returns a Promise, which is resolved elsewhere, it'd be hard to hone in on the specific `webpack` call to profile